src/components/embed/ui/atoms/icon/SuccessCheckSimple.tsx

```diff
@@ -0,0 +1,12 @@
+import type { SVGProps } from "react";
+const SvgSuccessCheckSimple = ({ width = 19, height = 19, ...props }: SVGProps<SVGSVGElement>) => (
+  <svg width={width} height={height} viewBox="0 0 19 19" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
```

Comment:
The casing should be camelCase when using SVGs as React component, otherwise you'll get a bunch of logs like this:

> react-dom.development.js:88  Warning: Invalid DOM property `clip-rule`. Did you mean `clipRule`?

Reply:
Addressed in 6461abde9245de3ed544bca4cf5f6646a097988e.